tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1932,4 +1932,45 @@ public function theGraphApiResponseShouldHaveTheRole(string $role): void {
 			. "\nExpected rolId for role '$role'' to be '" . $this->appEntity["appRoles"][$role] . "' but got '" . $response['appRoleId'] . "'"
 		);
 	}
+
+	/**
+	 * @When user :user gets details of the group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($groupName);
```

Comment:
@PrajwolAmatya i doubt have two methods to get group ID plz look over this and remove one of them if that is true https://github.com/owncloud/ocis/blob/2bf458385d7ffeb015066211e80b5eb33af299ee/tests/acceptance/features/bootstrap/FeatureContext.php#L4603-L4621  
https://github.com/owncloud/ocis/blob/2bf458385d7ffeb015066211e80b5eb33af299ee/tests/acceptance/features/bootstrap/Provisioning.php#L274-L303

Reply:
One of the method is now removed.